learning_resources/views_test.py

```diff
@@ -1463,20 +1463,7 @@ def test_course_run_problems_endpoint(client, user_role, django_user_model):
         reverse("lr:v0:tutorproblem_api-list-problems", args=[course_run.run_id])
     )
 
-    if user_role in ["admin", "group_tutor_problem_viewer"]:
```

Comment:
minor nit - the docstring for this test could be a bit more descriptive (include details about what is actually being tested like permissions and content of response etc)